kodilla-patterns/src/main/java/com/kodilla/patterns/builder/bigmac/Bigmac.java

```diff
@@ -0,0 +1,75 @@
+package com.kodilla.patterns.builder.bigmac;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public final class Bigmac {
+    private final Roll roll;
+    private final int burgers;
+    private final Sauce sauce;
+    private final List<Ingredients> ingredients;
+
+    public static class BigmacBuilder {
+        private Roll roll;
+        private int burgers;
+        private Sauce sauce;
+        private List<Ingredients> ingredients = new ArrayList<>();
+
+        public BigmacBuilder roll(Roll roll) {
+            this.roll = roll;
+            return this;
+        }
+
+        public BigmacBuilder burgers(int burgers) {
+            this.burgers = burgers;
+            return this;
+        }
+
+        public BigmacBuilder sauce(Sauce sauce) {
+            this.sauce = sauce;
+            return this;
+        }
+
+        public BigmacBuilder ingredient(Ingredients ingredient) {
+            ingredients.add(ingredient);
+            return this;
+        }
+
+        public Bigmac build() {
+            return new Bigmac(roll, burgers, sauce, ingredients);
```

Comment:
here you can add validation, for example: does not allow to create bigmac without burgers